tests/test_views.py

```diff
@@ -1,6 +1,8 @@
-from datetime import timedelta
+from datetime import datetime, timedelta
+from importlib import reload
 from unittest.mock import patch
 
+from rest_framework_simplejwt import serializers
```

Comment:
```suggestion
from datetime import timedelta
from importlib import reload
from unittest.mock import patch

from django.utils import timezone
from rest_framework_simplejwt import serializers
```

Use `django.utils.timezone` for datetime (avoid TZ naive datetime objs)